### PR TITLE
Renames Credential to unsecured payload and adds to terminology section

### LINKED ISSUE #45: Avoid and rename Credential based on SD-JWT
Try to find a new term.

### PR DESCRIPTION
**Pull request by awoie**
fixes #45

### CODE REVIEW

File: draft-terbu-sd-jwt-vc.md [hunk 1]
```diff
@@ -147,6 +147,11 @@ SD-JWT-based Verifiable Credential (SD-JWT VC):
 SD-JWT-based Verifiable Presentation (SD-JWT VP):
 : A Verifiable Presentation encoded using the Presentation format defined in [@!I-D.ietf-oauth-selective-disclosure-jwt].
 
+Unsecured payload of an SD-JWT VC:
+: A JSON object containing all selective and non-selective disclosable claims
+of the SD-JWT VC. The unsecured payload acts as the input JSON object to issue
+an SD-JWT VC complying to this specification.
+
```
Comment (danielfett) [inline draft-terbu-sd-jwt-vc.md:154]: I don't think we need to define this here if we only use it once. I suggest to move this down to the example section.

File: draft-terbu-sd-jwt-vc.md [hunk 2]
```diff
@@ -147,6 +147,11 @@ SD-JWT-based Verifiable Credential (SD-JWT VC):
 SD-JWT-based Verifiable Presentation (SD-JWT VP):
 : A Verifiable Presentation encoded using the Presentation format defined in [@!I-D.ietf-oauth-selective-disclosure-jwt].
 
+Unsecured payload of an SD-JWT VC:
+: A JSON object containing all selective and non-selective disclosable claims
```
Comment (danielfett) [inline draft-terbu-sd-jwt-vc.md:151]:
```suggestion
: A JSON object containing all selectively disclosable and non-selectively disclosable claims
```

File: draft-terbu-sd-jwt-vc.md (hunk 1 above)
Comment (awoie) [reply draft-terbu-sd-jwt-vc.md]: We are using the term in two sections, that is why I defined the term. But we can also remove it in the next version of the document if we don't think it adds any value.

**Comment by awoie** [comment]
@danielfett can we merge this now?

**Pull request merged**